libs/pages/layout/src/lib/feature/spotlight-trigger/spotlight-trigger.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { Spotlight } from '@qovery/shared/spotlight/feature'
+import { Icon, Kbd } from '@qovery/shared/ui'
+import { useFormatHotkeys } from '@qovery/shared/util-hooks'
+
+export function SpotlightTrigger() {
+  const [openSpotlight, setOpenSpotlight] = useState(false)
+  const metaKey = useFormatHotkeys('meta')
+  const { organizationId = '' } = useParams()
+
+  // Toggle the menu when ⌘K is pressed
+  useEffect(() => {
```

Comment:
Can we use `useEventListener` by usehooks ? https://usehooks.com/useeventlistener

Reply:
This hook is only available in experimental versions of `react` and `usehooks`